ubiconfig/utils/config_validation.py

```diff
@@ -4,10 +4,13 @@
 from jsonschema import validate
 
 DEFAULT_SCHEMA = os.path.join(os.path.dirname(__file__), "config_schema.json")
+""" Default yaml schema used for validation of UbiConfig configuration files """
 
 
 def validate_config(data, schema=None):
-    """validate the data according to the schema"""
+    """validate the data according to the schema
+
+If no schema is provided, :data:`DEFAULT_SCHEMA` is used"""
```

Comment:
Wrong ident?

Reply:
It's not wrong. Generated docs are same with indent or without it. Also for multiline string it's valid to be it like this. But yeah, I'll indent that more to have it prettier